ibis/backends/tests/test_aggregation.py

```diff
@@ -481,25 +481,37 @@ def mean_and_std(v):
             lambda t, where: t.double_col.std(how="sample", where=where),
             lambda t, where: t.double_col[where].std(ddof=1),
             id="std",
-            marks=[pytest.mark.notimpl(["druid"], raises=com.OperationNotDefinedError)],
+            marks=[
+                pytest.mark.notimpl(["druid"], raises=com.OperationNotDefinedError),
+                pytest.mark.notyet(["flink"], raises=AssertionError, strict=False),
```

Comment:
I don't know exactly what's happening here yet. It seems like `filter` syntax executes and returns the result, but the result is incorrect. `strict=False` is there because unfiltered versions of these functions produce the correct result.

Reply:
Fixed.